Sources/Core/API/Timeline.swift

```diff
@@ -48,8 +48,9 @@ public final class Timeline: Activatable {
 
     /// Repeat a closure by a given time interval, until it's cancelled by the returned token
```

Comment:
Maybe update the comment?
Other than that it looks good. Very useful!

Reply:
Yeah that's a good idea 👍

Reply:
Come to think of it, it would be cool if the CI bot was able to detect that a public API has been changed, and warn that you should update the docs 🤔